requirements.txt

```diff
@@ -10,3 +10,5 @@ six
 flask-testing
 gunicorn==20.0.4
 python-dotenv
+pytest
+pytest-cov
```

Comment:
Allows us to generate a coverage report. Readme already updated!

![](https://media3.giphy.com/media/3kFg2CD3uSTnopcbns/200w.gif?cid=5a38a5a2k50n0un0dz0mmtpw1xfz63k08vkkv8ygo0seeruv&rid=200w.gif)